examples/gallery/symbols/scatter.py

```diff
@@ -13,18 +13,20 @@
 import numpy as np
 import pygmt
 
-np.random.seed(19680801)
+rng = np.random.default_rng(seed=19680801)
 n = 200  # number of random data points
 
 fig = pygmt.Figure()
 fig.basemap(
-    region=[-0.1, 1.1, -0.1, 1.1],
+    region=[-1, 1, -1, 1],
     projection="X10c/10c",
-    frame=["xa0.2fg", "ya0.2fg", "WSrt"],
+    frame=["xa0.5fg", "ya0.5fg", "WSrt"],
 )
 for fill in ["gray73", "darkorange", "slateblue"]:
-    x, y = np.random.rand(2, n)  # random X and Y data in [0,1]
-    size = np.random.rand(n) * 0.5  # random size [0,0.5], in cm
+    x = rng.normal(0, 0.5, n)  # random x data
+    y = rng.normal(0, 0.5, n)  # random y data
```

Comment:
```suggestion
    x = rng.normal(loc=0, scale=0.5, size=n)  # random x data
    y = rng.normal(loc=0, scale=0.5, size=n)  # random y data
```

Reply:
I also updated this for `size` in the next line.